File: CS/04_TableAndChart/SaveChartAsImage.cs
using Spire.Presentation;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SaveChartAsImage
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //create PPT document 
            Presentation presentation = new Presentation();

            //load PPT file from disk
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\Chart.pptx");

            //Save chart as image in .Png format
            Image image = presentation.Slides[0].Shapes.SaveAsImage(0);
            image.Save("Chart.png", System.Drawing.Imaging.ImageFormat.Png);

            System.Diagnostics.Process.Start("Chart.png");
        }
    }
}

File: VB/04_TableAndChart/SaveChartAsImage.vb
Imports Spire.Presentation
Imports System.ComponentModel
Imports System.Text

Namespace SaveChartAsImage
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'create PPT document 
			Dim presentation As New Presentation()

			'load PPT file from disk
			presentation.LoadFromFile("..\..\..\..\..\..\Data\Chart.pptx")

			'Save chart as image in .Png format
			Dim image As Image = presentation.Slides(0).Shapes.SaveAsImage(0)
			image.Save("Chart.png", System.Drawing.Imaging.ImageFormat.Png)

			Process.Start("Chart.png")
		End Sub
	End Class
End Namespace
